core/src/main/scala/eu/shiftforward/apso/io/LocalFileDescriptor.scala

```diff
@@ -116,7 +116,11 @@ case class LocalFileDescriptor(initialPath: String) extends FileDescriptor with
     }
   }
 
-  def stream() = new FileInputStream(file)
+  def stream(offset: Long = 0L) = {
+    val is = new FileInputStream(file)
+    if (offset > 0L) is.skip(offset)
```

Comment:
Does `FileInputStream` deal with this operation correctly?

Reply:
The `FileInputStream` skip is a `native` method. I assume that it will behave as expected, but it can depend on multiple factors.

According to https://stackoverflow.com/questions/3669598/does-fileinputstream-skip-do-a-seek, OpenJDK does a seek, so this should be fine.